src/components/UI/headings/PageSection.tsx

```diff
@@ -0,0 +1,19 @@
+import { Heading, HeadingProps } from '@chakra-ui/react';
+import { FC } from 'react';
+
+export const PageSection: FC<HeadingProps> = ({ children, ...props }) => {
+  return (
+    <Heading
+      as='h3'
+      color='#e44550'
+      fontSize='22px'
+      fontWeight={700}
+      lineHeight='29px'
+      className='maison-neue-mono'
```

Comment:
Just an idea. Perhaps we could define this in the theme to keep everything scoped there and avoid creating custom css classes.

We could do something like:
```ts
extendTheme({
  ...
  components: {
    Heading: {
      variants: {
        'page-section': {
          fontFamily: 'Maison Neue Mono'
        }
      }
    }
  }
});

// and then in here, just
<Heading variant="page-section" ...>
```

Reply:
Fixed! I wasn't able to make it work [using a Font component](https://chakra-ui.com/guides/recipes/using-fonts#option-2-using-font-face) like the Chakra guide suggests, but the variant works with the font-face declared on the `global.css` file.

Reply:
Great! gj